patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -792,21 +792,30 @@
 +     */
 +    public void onHorseArmorTick(World world, net.minecraft.entity.EntityLiving horse, ItemStack armor) {}
 +    
++    @SideOnly(Side.CLIENT)
++    private net.minecraft.client.renderer.tileentity.TileEntityItemStackRenderer teisr = null;
```

Comment:
`@Nullable`

Reply:
Is it necessary?  The field is initialized to be null, and it's private...  It'd be an extra line with no real purpose.

Reply:
IDEs will complain, might as well make it nice?